gulpfile.js

```diff
@@ -0,0 +1,44 @@
+// Include gulp
+var gulp = require('gulp');
+
+// Include plugins
+var concat = require('gulp-concat');
+var uglify = require('gulp-uglify');
+var rename = require('gulp-rename');
+var sass = require('gulp-sass');
+var minifyCSS = require("gulp-minify-css");
+
+
+// Concatenate & Minify JS
+gulp.task('src-js', function () {
+	return gulp.src('src/js/**/*.js')
+		.pipe(concat('main.min.js'))
+		//.pipe(uglify()) //TODO: uncomment it when finish
+		.pipe(gulp.dest('dist/js'));
+});
+
+// Move libs
+gulp.task('move-libs', function () {
+	return gulp.src('bower_components/**/**.min.js')
```

Comment:
all js files which you take from bower_components folder must be merged into one file